www/_template/reference/configuration.md

```diff
@@ -124,211 +337,239 @@ module.exports = {
+**Type**: `string`
 
-#### buildOptions.sourceMaps | `boolean` | Default: `false`
+**Default**: `__snowpack__`
 
-- **_Experimental:_** Set to `true` to enable source maps
+Rename the default directory for Snowpack metadata.
 
-#### buildOptions.webModulesUrl | `string` | Default: `web_modules`
+Snowpack-related metadata includes [HMR](/concepts/hot-module-replacement) and [ENV](/reference/configuration#environment-variables) info.
 
-- Rename your web modules directory.
+### buildOptions.minify
 
-#### buildOptions.jsxFactory | `string` | Default: `React.createElement` (or `h` if Preact import is detected)
+**Type**: `boolean`
 
-- Set the name of the used function to create JSX elements.
+**Default**: `true`
 
-#### buildOptions.jsxFragment | `string` | Default: `React.Fragment` (or `Fragment` if Preact import is detected)
+By default, Snowpack will minify your dependencies on `snowpack build` using Esbuild. TODO is this correct?
 
-- Set the name of the used function to create JSX fragments.
+### buildOptions.sourceMaps
 
-### config.testOptions
+**Type**: `boolean`
 
-#### testOptions.files | `string[]` | Default: `["__tests__/**/*", "**/*.@(spec|test).*"]`
+**Default**: `false`
 
-- The location of all test files.
-- All matching test files are scanned for installable dependencies during development, but excluded from both scanning and building in your final build.
+Generates source maps.
 
-### config.proxy
+**_Experimental:_** Lit Element, Preact, Svelte and React sourcemaps in build and dev. With blank templates, esbuild will generate source maps if it can, if another compiler like Babel, Vue, or Svelte isn’t being used
 
-If desired, `"proxy"` is where you configure the proxy behavior of your dev server. Define different paths that should be proxied, and where they should be proxied to.
+TODO: is this correct?
 
-The short form of a full URL string is enough for general use. For advanced configuration, you can use the object format to set all options supported by [http-proxy](https://github.com/http-party/node-http-proxy).
+### buildOptions.watch
+**Type**: `boolean`
 
-This configuration has no effect on the final build.
+**Default**: `false`
 
-Example:
+Run Snowpack's build pipeline through a file watcher.
 
-```js
-// snowpack.config.json
-{
-  "proxy": {
-    // Short form:
-    "/api/01": "https://pokeapi.co/api/v2/",
-    // Long form:
-    "/api/02": {
-      on: { proxyReq: (p, req, res) => /* Custom event handlers (JS only) */ },
-      /* Custom http-proxy options */
-    }
-  }
-}
-```
+Designed to allow local development with an output directory that can be served by a custom frontend server (ex: Rails, PHP, etc.)
 
-Options:
+TODO: is this right? based on https://github.com/snowpackjs/snowpack/issues/1002
```

Comment:
@FredKSchott wrote this for `buildOptions.watch` based on this issue, lmk if correct

Reply:
cleaned up